pyhealth/models/cross_modal_transformer.py

```diff
@@ -0,0 +1,847 @@
+        >>> val_loader = get_dataloader(val_dataset, batch_size=32, shuffle=False)
+        >>> test_loader = get_dataloader(test_dataset, batch_size=32, shuffle=False)
+
+        
+        >>> from pyhealth.models import Seq_Cross_Modal_Transformer_PyHealth
+        >>> model = Seq_Cross_Modal_Transformer_PyHealth( dataset= sleepedf_task_ds, 
+        ...                                    feature_keys= ['signal'], 
+        ...                                    label_key= ['label'], 
+        ...                                    mode= 'multiclass', 
+        ...                                    d_model = 128,
+        ...                                    num_epoch_seq = num_epoch_seq,
+        ...                                    dim_feedforward=512,
+        ...                                    window_size = 50,
+        ...                                    num_classes = 6,).to("cuda:0")
+        >>> # data batch
+        >>> data_batch = next(iter(train_loader))
+        >>> # try the model
+        >>> ret = model(**data_batch)
+        >>> print(ret)
+        >>> #try loss backward
+        >>> ret["loss"].backward()
+
+    '''
+
+    def __init__(self, 
+                 dataset: BaseSignalDataset, 
+                 feature_keys: List[str], 
+                 label_key: str, 
+                 mode: str, 
+                 d_model = 128,
+                 num_epoch_seq = 5,
+                 dim_feedforward=512,
+                 window_size = 50,
+                 num_classes = 6,
+                 **kwargs):
+        super().__init__(dataset, feature_keys, label_key, mode)
+
+    
+        self.num_epoch_seq = num_epoch_seq
+        self.num_classes = num_classes
+        
+        self.seq_cross_modal_transformer = Seq_Cross_Modal_Transformer_Network(d_model = d_model,
+                                                                                num_epoch_seq = num_epoch_seq,
+                                                                                dim_feedforward=dim_feedforward,
+                                                                                window_size = window_size,
+                                                                                num_classes = num_classes)
+        
+    def get_sliced_seq(self, signal,epoch_seconds = 30,sample_rate = 100):
+        '''
+        Slice the signal into 30s epochs
+        Args:
+            signal: Tensor, input signal
+            epoch_seconds: int, length of the epoch
+            sample_rate: int, sampling rate of the signal
+        Returns:
+            Tensor, sliced signal
+        '''
+        sample_length = sample_rate * epoch_seconds
+        sliced_seq = []
+        for slice_index in range(signal.shape[-1] // sample_length):
+            sliced_seq.append(signal[:,:,slice_index * sample_length : (slice_index + 1) * sample_length])
+        return torch.cat(sliced_seq,dim = 1)
+    
+    def forward(self,**kwargs):
```

Comment:
Please also handle the kwarg `embed` - see https://github.com/sunlabuiuc/PyHealth/blob/89ee19fa084f4cc0be8205eb59b82375ff9e2c69/pyhealth/models/retain.py#L364. This is for the calibration module. Thanks!